eva/server/command_handler.py

```diff
@@ -63,7 +81,8 @@ def handle_request(transport, request_message):
         logger.warn(e)
         response = Response(status=ResponseStatus.FAIL, batch=None, error=str(e))
     else:
-        response = Response(status=ResponseStatus.SUCCESS, batch=output_batch)
+        response = Response(status=ResponseStatus.SUCCESS,
```

Comment:
Looks like you have a linter setting with a different max line length. Black uses 88